packages/contracts-bedrock/src/L1/BatchInbox.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+import {IBatchVerifier} from "interfaces/L1/IBatchVerifier.sol";
+
+contract BatchInbox {
+    IBatchVerifier immutable batchVerifier;
+
+    constructor(IBatchVerifier _batchVerifier) {
+        batchVerifier = _batchVerifier;
+    }
+
+    fallback() external {
+        if (blobhash(0) != 0) {
```

Comment:
Is this to support [EIP4844](https://www.eip4844.com/) ?

Reply:
Yes, OP batcher uses blob transactions in some configurations.